src/main/java/leelawatcher/gui/LeelaWatcher.java

```diff
@@ -113,8 +131,18 @@ public void windowClosing(WindowEvent e) {
     splitPane.setLeftComponent(boardView);
     textScrollPane = new JScrollPane();
     splitPane.setRightComponent(textScrollPane);
-    leelaOutputTextArea = new JTextArea();
-    leelaOutputTextArea.setRows(8);
+
```

Comment:
You didn't read the comment at the top of this method did you :) ? This method must not be edited. it is generated by a tool in my IDE. This code needs to move to createUIComponents() or main() before the call to frame.pack() ... Also, pls leave the rows > 4 for the text area so one has a chance to see the game ending message.